apps/server/src/shared/infra/rabbitmq/rpc-message-producer.ts

```diff
@@ -0,0 +1,37 @@
+import { AmqpConnection } from '@golevelup/nestjs-rabbitmq';
+import { ErrorMapper } from './error.mapper';
+import { RpcMessage } from './rpc-message';
+
+export abstract class RpcMessageProducer {
```

Comment:
AmqpRpcClient

Reply:
We leave the designations so that you have a clear distinction between the type of clients